004-quarkus-HHH-and-HV/src/main/resources/import.sql

```diff
@@ -0,0 +1,6 @@
+insert into account (id, email) values (1, 'foo@bar.com');
```

Comment:
`BEGIN TRANSACTION;` is missing imho

Reply:
I could it reproduce the data loss issue without the BEGIN TRANSACTION.

Reply:
it's intentional to omit BEGIN TRANSACTION to reproduce the issue

Reply:
Can we have commented out line with note about that so it's easy to have context 

```
-- BEGIN TRANSACTION commented out intentionally to replicate the original issue
```

Reply:
we removed the `BEGIN TRANSACTION` statement because regardless having it or not, we could reproduce the original issue. It was an unnecessary line, so I don't think it's needed any comment. wdyt?

Reply:
ok